### ISSUE TITLE
General status

### ISSUE THREAD

**Issue by itsdfish** [spm/SPM.jl#2]
Hello, 

Thank you for putting this package together. Its encouraging to see that some effort has been made in making SPM available in an open source language. I think Julia is a good choice. Out of curiosity, is SPM.jl feature complete, and would it be possible to use/learn along with the documentation? From what I can tell, the manual is geared towards Matlab and its GUI interface. 

Thank you.

**Comment by aTrotier** [comment]
I don't think you have access to all the SPM functionality like the GM/WM segmentation.
It would be great to be able to have a PURE julia package without a ccall to some library :)
I will be happy to participate !

**Comment by JohnAshburner** [comment]
There is not yet much translation of SPM work into Julia.  What has been done so far was just to explore Julia's potential. How much will be done in future will depend on the resources available to the SPM developer team.

**Comment by itsdfish** [comment]
Understandable. It might be a good idea to make a note in the README file that it is not a complete re-implementation.
